src/main/java/com/reckue/post/controllers/NodeController.java

```diff
@@ -69,15 +90,32 @@ public NodeResponse update(@PathVariable String id, @RequestBody @Valid NodeRequ
      */
     @GetMapping
     public List<NodeResponse> findAll(@RequestParam(required = false) Integer limit,
-                                         @RequestParam(required = false) Integer offset,
-                                         @RequestParam(required = false) String sort,
-                                         @RequestParam(required = false) Boolean desc) {
+                                      @RequestParam(required = false) Integer offset,
+                                      @RequestParam(required = false) String sort,
+                                      @RequestParam(required = false) Boolean desc) {
 
         return nodeService.findAll(limit, offset, sort, desc).stream()
                 .map(NodeConverter::convert)
                 .collect(Collectors.toList());
     }
 
+    /**
+     * This type of request allows to get all the objects by user id, process it using the converter.
+     *
+     * @param userId user identifier
+     * @param limit  quantity of objects
+     * @param offset quantity to skip
+     * @return list of objects of class NodeResponse
+     */
+    @GetMapping("/userId/{userId}")
```

Comment:
Maybe it would be better to call it `/user/{id}`?

Reply:
maybe ```user/{userId}```? i think it must be exactly "userId" because others endpoints are connected with id of concrete model